typings/resin-sdk.d.ts

```diff
@@ -259,6 +259,16 @@ declare namespace ResinSdk {
 		owns__device: ReverseNavigationResource<Device>;
 		owns__release: ReverseNavigationResource<Release>;
 		is_depended_on_by__application: ReverseNavigationResource<Application>;
+		user__is_member_of__application: ReverseNavigationResource<
+			ApplicationMember
+		>;
+	}
+
+	interface ApplicationMember {
```

Comment:
This was `Collaborator` in the UI until now, so I guess `ApplicationMember` is sensible and not inconveniently long.
I would love suggestions though.

Reply:
`ApplicationMember` is fine I think